app/components/DR/ButtonUpDrawer.js

```diff
@@ -9,6 +10,8 @@ import {
   View,
 } from 'react-native';
 
+import languages from '../../locales/languages';
+
 const { width, height } = Dimensions.get('window');
 
 export default class BottomUpPanel extends Component {
```

Comment:
After changing languages to **withTranslation** remove the **export default** form here and add it at the end of the file like this `export default withTranslation()(BottomUpPanel)`
```suggestion
class BottomUpPanel extends Component {
```

Reply:
Does not work

Reply:
It did not work